plugins/org.jkiss.dbeaver.tasks.ui/src/org/jkiss/dbeaver/tasks/ui/wizard/TaskConfigurationWizard.java

```diff
@@ -221,6 +222,13 @@ && getContainer().getCurrentPage() != activeWizardPage
                 && !page.getControl().isDisposed()
             ) {
                 activeWizardPage.updatePageCompletion();
+                if (activeWizardPage.getErrorMessage() != null
+                    && (getContainer().getErrorMessage() == null
+                    || activeWizardPage.getPreviousPage().equals(wizardWithError))) {
```

Comment:
Why are you checking only previous page?

Reply:
Ok let's check all previous pages

Reply:
nvm, the pages are already sorted, so I will remove this condition